### PR TITLE
Fix MSP receiver control page

### PR DESCRIPTION
**Pull request by McGiverGim**
The MSP receiver control page is broken. This fixes https://github.com/betaflight/betaflight-configurator/issues/922

I've made a hack to use the i18n object of the opener page, but the `localizePage()` method does not work, so I made the localization by my own. Maybe there's a better way, but now I can't look into it and this fixes the problem.

### CODE REVIEW

File: locales/en/messages.json [hunk 1]
```diff
@@ -1401,6 +1401,13 @@
         "message": "Preview"
     },
 
+    "receiverMspWarningText": {
+        "message": "These sticks allow BetaFlight to be armed and tested without a transmitter or receiver being present. However, <strong>this feature is not intended for flight and propellers must not be attached.</strong><br><br>This feature does not guarantee reliable control of your craft. <strong>Serious injury is likely to result if propellers are left on.</strong>"
```
Comment (mikeller) [inline locales/en/messages.json]: The commonly used for is 'Betaflight', not 'BetaFlight'.

Comment (McGiverGim) [reply locales/en/messages.json]: True. This was the original text. I will modify it after the weekend. I'm out.

**Comment by mikeller** [comment]
FTFY.

**Comment by McGiverGim** [comment]
Perfect! Thanks!

**Pull request merged**